OpenKh.Tools.BbsMapStudio/Program.cs

```diff
@@ -0,0 +1,69 @@
+using McMaster.Extensions.CommandLineUtils;
+using OpenKh.Tools.Common.CustomImGui;
+using System;
+using System.Reflection;
+
+namespace OpenKh.Tools.BbsMapStudio
+{
+    [Command("OpenKh.Tools.BbsMapStudio")]
+    [VersionOptionFromMember("--version", MemberName = nameof(GetVersion))]
+    class Program : IDisposable
+    {
+        [STAThread]
+        static int Main(string[] args)
+        {
+            return CommandLineApplication.Execute<Program>(args);
+        }
+
+        protected int OnExecute(CommandLineApplication app)
+        {
+            Run();
+            return 0;
+        }
+
+        #region CommandLine
+        private static string GetVersion()
+            => typeof(Program).Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion;
+
+        [Argument(0, "KH2 GamePath")]
```

Comment:
You probably meant BBS GamePath ;)

Reply:
Yeah, I should probably change that. Thanks.

Reply:
I fixed this.